src/js/modules/infragistics.ui.editors.js

```diff
@@ -7788,21 +7788,37 @@
 				spin type="string" Spin buttons are located on the right side of the editor
 			*/
 			buttonType: "none",
-			/* type="number" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
+			/* type="number|object" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
 				When not editing (focused) the delta is applied on the day if available in the input mask or the lowest available period.
 				When in edit mode the time period, where the cursor is positioned, is incremented or decremented with the defined delta value.
-				The value can be only a positive integer number, otherwise it will be set as 1, or in the cases with double or float the the whole part will be taken.
+				If spinDelta is defined as an intereger, then this value it is applied to all time periods - years, days, minutes, etc.
+				If spinDelta is defined as an object, then each time period can be defined with specific values. If in that case, some of the time periods don't have values, 1 is set as default one.
+				The value of each deltas can be only a positive integer number, otherwise exception will be thrown, or in the cases with double or float the the whole part will be taken.

```

Comment:
"Accepted values for deltas are positive integer numbers, and the fractional portion of floating point numbers is ignored"
This should stay as a general note I think.